.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        env: [py37_iris30, py38_iris30]
+        env: [environment_a, environment_b, latest, conda-forge]
```

Comment:
If I understand this correctly, it means that future PRs will now have to pass with "latest", that is, all package updates, as well as with the environments that we actually want to support. I don't think this is a good idea and will interrupt our development process.

Reply:
I've removed the no-longer-used data from `EXPECTED` in 9c8b238.